internal/genny/assets/webpack/templates/dot-yarnrc.yml.tmpl

```diff
@@ -0,0 +1,4 @@
+enableGlobalCache: true
+logFilters:
```

Comment:
Could you please consider using `yarn config set` instead of this template? It doesn't help reducing 4k lines of `yarn install` logs if home version is not berry.

Reply:
Ok I see now. Yarn classic will only look for `.yarnrc` not `.yarnrc.yml`, so using `yarn config set` will work on both versions.

Reply:
Updated in https://github.com/gobuffalo/cli/pull/96/commits/4858a22a319fa7d641d87356c4112c9ae8c60488. 🤞 that solves everything